doc/write_your_own/components/Workflow1.ipynb

```diff
@@ -86,18 +57,7 @@
     }
    },
    "source": [
-    "Create cube picker consiting of a text acceptor for the file path and a multiple selection pane for the cube selector"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "urth": {
-     "dashboard": {}
-    }
-   },
-   "source": [
-    "Please enter the path of your cube(s) and then select the cube or cubelist you would like to plot with."
+    "Create cube picker consiting of a text input box for the file path and a selection widget for the cube selector."
```

Comment:
Again, it's outside of the limits of the changes here, but you're using more than one `h1` (a single # prefixing a line of markdown in a text cell) in this notebook. It's bad form to have more than one `h1` in any webpage, including jupyter notebooks (it makes navigation of the webpage difficult, especially when using accessibility tools).